lib/src/analyzers/unused_files_analyzer/reporters/reporters_list/console/unused_files_console_reporter.dart

```diff
@@ -26,14 +28,18 @@ class UnusedFilesConsoleReporter
 
     for (final analysisRecord in sortedRecords) {
       output.writeln(
-        '${warnigPen('⚠')} unused file: ${analysisRecord.relativePath}',
+        '${warningPen('⚠')} unused file: ${analysisRecord.relativePath}',
       );
     }
 
+    final removeFilesSuffix = additionalParams?.deleteUnusedFiles ?? false
+        ? ', all unused files were successfully deleted'
+        : '';
+
     output
       ..writeln('')
       ..writeln(
-        '${alarmPen('✖')} total unused files - ${alarmPen(sortedRecords.length)}',
+        '${alarmPen('✖')} total unused files - ${alarmPen(sortedRecords.length)}$removeFilesSuffix',
```

Comment:
@incendial if we remove all unused files, why we in alarm state? I think this is good or info state, our repo not contains unused files.

Reply:
Fixed